### PR TITLE
Update the SDK for the API's we added in swagger (P1's)

### PR DESCRIPTION
**Pull request by shimedh**
**API's added:**
1]. POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Logic/workflows/{flowName}/listSwagger
2]. POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Logic/workflows/{flowName}/regenerateAccessKey
3]. POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Logic/workflows/{flowName}/versions/{flowVersion}/triggers/{triggerName}/listCallbackUrl
4]. POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Logic/workflows/{flowName}/triggers/{triggerName}/histories/{historyName}/resubmit

**Updated structure** - "WorkflowRunActionProperties"

**azure-rest-api-specs Pull requests:**
https://github.com/Azure/azure-rest-api-specs/pull/1102
https://github.com/Azure/azure-rest-api-specs/pull/1118

**Latest swagger spec:**
https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e7adb40af3a97889b42a18377fda13c91ad8a5db/arm-logic/2016-06-01/swagger/logic

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shimedh** [comment]
@azuresdkci What should I do with the above merge conflicts? I don't see any locally or going to the "Files Changed" tab above.

**Comment by shimedh** [comment]
Updated the PR after resolving the conflicts

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/SDKs/Logic/Logic.Tests/project.json [hunk 1]
```diff
@@ -37,6 +37,6 @@
     "Microsoft.Azure.ResourceManager": "1.0.0-preview",
     "xunit": "2.2.0-beta2-build3300",
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
-    "Microsoft.Azure.Management.Logic": "2.0.0"
+    "Microsoft.Azure.Management.Logic": "3.1.0"
```
Comment (shahabhijeet) [inline src/SDKs/Logic/Logic.Tests/project.json]: @shimedh this file actually needs to be deleted. We no longer use this file

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj [hunk 2]
```diff
@@ -174,6 +174,12 @@
     <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ValidateWorkflow.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ListSwagger.json">
```
Comment (shahabhijeet) [inline src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj]: @shimedh no need to add individual files.
Rather we should get the entire directory contents
<None Update="SessionRecords\**\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>    
    </None>

Comment (shahabhijeet) [inline src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj]: I also do not see version bump for this change.
Update the version in .csproj file as well as AssemblyInfo.cs

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**